fabfile.py

```diff
@@ -102,7 +83,6 @@ def _pre_check():
 
 def _post_check():
     if "Darwin" in system():
-        _ensure_solium()
         _ensure_ganache_cli()
```

Comment:
Not part of this story, but it would be great if you also deleted the ganache references from here, thanks!

Reply:
I just pushed a new commit to remove Ganache 👍 :)